### PR TITLE
possible fix for ssr handling of api calls

### PR DESCRIPTION
**Pull request by internettrans**

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -1,16 +1,50 @@
-import fetch from "unfetch";
+import fetch from "isomorphic-unfetch";
+import https from "https";
+
+const nodeGetPokemons = () => {
+  return new Promise((resolve, reject) => {
+    https
```
Comment (internettrans) [inline src/api.js:6]: the https module only works in Node, not in the browser.

Comment (tsukhu) [reply src/api.js:6]: Actually it's being called only for the node piece and for the browser we still use fetch

File: src/api.js [hunk 2]
```diff
@@ -1,16 +1,50 @@
-import fetch from "unfetch";
+import fetch from "isomorphic-unfetch";
+import https from "https";
```
Comment (internettrans) [inline src/api.js:2]: This import should fail in the browser build

**Review by internettrans — COMMENTED** [review]
We can try this fix - I don't like the hackiness of it, but it's better to have this example in a working state.

**Comment by tsukhu** [comment]
> We can try this fix - I don't like the hackiness of it, but it's better to have this example in a working state.

Totally agreed, this too is not perfect as we are still calling the client fetch as well

**Pull request merged**